codespell_lib/data/dictionary.txt

```diff
@@ -11401,6 +11404,7 @@ puchasing->purchasing
 Pucini->Puccini
 Puertorrican->Puerto Rican
 Puertorricans->Puerto Ricans
+puffer->buffer
```

Comment:
Remove this, it's a perfectly valid word.

Reply:
Done